src/useStore.ts

```diff
@@ -1,212 +1,29 @@
   isCopilotLspServerReady: boolean
   setIsKclLspServerReady: (isKclLspServerReady: boolean) => void
   setIsCopilotLspServerReady: (isCopilotLspServerReady: boolean) => void
-  buttonDownInStream: number | undefined
-  setButtonDownInStream: (buttonDownInStream: number | undefined) => void
-  didDragInStream: boolean
-  setDidDragInStream: (didDragInStream: boolean) => void
-  fileId: string
-  setFileId: (fileId: string) => void
-  streamDimensions: { streamWidth: number; streamHeight: number }
-  setStreamDimensions: (dimensions: {
-    streamWidth: number
-    streamHeight: number
-  }) => void
   setHtmlRef: (ref: React.RefObject<HTMLDivElement>) => void
   htmlRef: React.RefObject<HTMLDivElement> | null
-
-  showHomeMenu: boolean
-  setHomeShowMenu: (showMenu: boolean) => void
-  openPanes: SidebarType[]
-  setOpenPanes: (panes: SidebarType[]) => void
-  homeMenuItems: {
-    name: string
-    path: string
-  }[]
-  setHomeMenuItems: (items: { name: string; path: string }[]) => void
 }
 
-export const useStore = create<StoreState>()(
-  persist(
-    (set, get) => {
-      return {
-        setMediaStream: (mediaStream) => set({ mediaStream }),
-        isStreamReady: false,
-        setIsStreamReady: (isStreamReady) => set({ isStreamReady }),
-        isKclLspServerReady: false,
-        isCopilotLspServerReady: false,
-        setIsKclLspServerReady: (isKclLspServerReady) =>
-          set({ isKclLspServerReady }),
-        setIsCopilotLspServerReady: (isCopilotLspServerReady) =>
-          set({ isCopilotLspServerReady }),
-        buttonDownInStream: undefined,
-        setButtonDownInStream: (buttonDownInStream) => {
-          set({ buttonDownInStream })
-        },
-        setHtmlRef: (htmlRef) => {
-          set({ htmlRef })
-        },
-        htmlRef: null,
-        didDragInStream: false,
-        setDidDragInStream: (didDragInStream) => {
-          set({ didDragInStream })
-        },
-        // For stream event handling
-        fileId: '',
-        setFileId: (fileId) => set({ fileId }),
-        streamDimensions: { streamWidth: 1280, streamHeight: 720 },
-        setStreamDimensions: (streamDimensions) => {
-          set({ streamDimensions })
-        },
-
-        // tauri specific app settings
-        defaultDir: {
-          dir: '',
-        },
-        openPanes: ['code'],
-        setOpenPanes: (openPanes) => set({ openPanes }),
-        showHomeMenu: true,
-        setHomeShowMenu: (showHomeMenu) => set({ showHomeMenu }),
-        homeMenuItems: [],
-        setHomeMenuItems: (homeMenuItems) => set({ homeMenuItems }),
-      }
+export const useStore = create<StoreState>()((set, get) => {
+  return {
+    setIsStreamReady: (isStreamReady) => set({ isStreamReady }),
+    setIsKclLspServerReady: (isKclLspServerReady) =>
+      set({ isKclLspServerReady }),
+    setIsCopilotLspServerReady: (isCopilotLspServerReady) =>
+      set({ isCopilotLspServerReady }),
+    setHtmlRef: (htmlRef) => {
+      set({ htmlRef })
```

Comment:
whoa this is so little left i can likely help on the lsp side but i also have no idea what im doing but basically there are callbacks on starting those

Reply:
Yeah, thanks, I was planning on getting playwright happy enough and then running the 4 of these bits of state past you and Frank, because I have some ideas but not sure.

Reply:
that sounds good to me i opened a different pr for the lsp one that was literally not using this

Reply:
Okay sweet, I'll touch base with Frank about the others.